client/reader/stream/post.jsx

```diff
@@ -171,6 +171,13 @@ export default class ReaderPostCardAdapterFluxContainer extends React.Component
 		FeedPostStore.off( 'change', this.updateState );
 	}
 
+	shouldComponentUpdate( nextProps, nextState ) {
+		const shouldUpdate = this.props !== nextProps ||
```

Comment:
> ah, yeah, that's true, but each render of the component will create a new props object. 

where are you getting that from?  Not opposed to adding it, but I haven't seen that happening in my testing